btcpay-backup.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash -e
+
+# Please be aware of these important issues:
+#
+# - Old channel state is toxic and you can loose all your funds, if you or someone
+#   else closes a channel based on the backup with old state - and the state changes
+#   often! If you publish an old state (say from yesterday's backup) on chain, you
+#   WILL LOSE ALL YOUR FUNDS IN A CHANNEL, because the counterparty will publish a
+#   revocation key!
+
+if [ "$(id -u)" != "0" ]; then
+  echo "This script must be run as root."
+  echo "Use the command 'sudo su -' (include the trailing hypen) and try again"
+  exit 1
+fi
+
+# preparation
+docker_dir=/var/lib/docker
```

Comment:
Can we avoid hard coding this?

Reply:
Done in [030faa7](https://github.com/btcpayserver/btcpayserver-docker/pull/641/commits/030faa7717746ca7210a30be67df68916db9a77a).